test/Microsoft.AspNet.Mvc.Core.Test/Rendering/BufferEntryCollectionTest.cs

```diff
@@ -1,160 +0,0 @@
-﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
```

Comment:
this should be removed immediately, separate from this PR.  the `BufferEntryCollection` class is already in Common and a reaction PR to use it in MVC shouldn't be mixed in with a reaction to aspnet/MVC#312.